src/Compile/CompilePanel.ts

```diff
@@ -154,12 +166,114 @@ export class CompilePanel {
       <head>
```

Comment:
in the future, this could be extracted to
```js
  return getHTML(toolkitUri, jsUri, cssUri);
```

Reply:
Yes. I left similar opinion as a `TODO` at L160. 😸

Reply:
👍